src/main/java/cloud/eppo/logging/Assignment.java

```diff
@@ -0,0 +1,109 @@
+package cloud.eppo.logging;
+
+import cloud.eppo.Utils;
+import cloud.eppo.ufc.dto.SubjectAttributes;
+import java.util.Date;
+import java.util.Map;
+
+/**
+ * @noinspection unused
+ */
+public class Assignment {
```

Comment:
Mostly not new code; Largely based on the Android implementation ([source](https://github.com/Eppo-exp/android-sdk/blob/main/eppo/src/main/java/cloud/eppo/android/logging/Assignment.java#L12)).